src/Elastic.Apm/Api/IMetricSet.cs

```diff
@@ -16,6 +16,7 @@ public interface IMetricSet
 		/// </summary>
 		IEnumerable<MetricSample> Samples { get; set; }
 
+		// TODO: Rename to Timestamp for consistency?
```

Comment:
This interface is public, but so far we don't expose anything to collect custom metrics, so if we rename, then I'd say we should do it soon and not wait until this may be exposed more.

Reply:
++ I'll open another PR to address